src/engine/basics/Settings.cxx

```diff
@@ -9,6 +9,25 @@
 
 #include <iomanip>
 
+class sax_no_exception : public nlohmann::detail::json_sax_dom_parser<json>
```

Comment:
What is this? What does it do?

Reply:
this code for extended error information, like you forgot comma in line 115
without this you just see in log, settings.json have error

Reply:
i didn't know this eists. sounds good! have't tried it yet though